### ISSUE TITLE
CKE5 problem: code styles not available

### ISSUE THREAD

**Issue by argiepiano** [backdrop-ops/forum.backdropcms.org#148]
Since a couple of days ago, the Forum now uses CKE5. The problem I'm seeing is that it's impossible to select the Styles: Code block. Those are grayed out:
<img width="749" alt="Screen Shot 2024-03-09 at 8 18 54 AM" src="https://github.com/backdrop-ops/forum.backdropcms.org/assets/9938978/beec1746-1d3c-4351-acad-00970cc5c229">

And if you choose the PHP style, this is equivalent to the old "inline PHP" style, and the result includes blank lines in between each line of code.
<img width="718" alt="Screen Shot 2024-03-09 at 8 21 21 AM" src="https://github.com/backdrop-ops/forum.backdropcms.org/assets/9938978/5699fc06-cf62-495e-83ed-1705f4ced882">

In short, with CKE5 it has become impossible to paste well-formatted code.

BTW: is there any way to use markdown in the forum. This would be A-MA-ZING!

**Comment by olafgrabienski** [comment]
> In short, with CKE5 it has become impossible to paste well-formatted code.

There's a related report in a forum comment, see https://forum.backdropcms.org/forum/snazzy-theme-help-needed-photo-not-displaying-correctly?page=0%2C0#comment-6731.

**Comment by argiepiano** [comment]
There is a workaround to my message above - you can select Markdown as one of the text styles. This allows you do use markdown in the message. However, I don't know if there is a workaround for the forum comment you pointed out.
